tests/cli/StopOnFailureCest.php

```diff
@@ -0,0 +1,27 @@
+<?php
+namespace Robo;
+
+use \CliGuy;
+
+class StopOnFailureCest
+{
+    public function _before(CliGuy $I)
+    {
+        $I->amInPath(codecept_data_dir('sandbox'));
+    }
+
+    public function parallelProcessesGetStoppedIfStopOnFailIsSet(CliGuy $I)
```

Comment:
Nice test. If `posix_setpgid` works, make a second test just like this one that does not use `exec`, and markTestSkipped if the posix commands are not available.

Reply:
I'm sorry, I just realized I've never read you change request: can you explain me what you are asking for? I don't get it :confused:

Reply:
I added a summary of the requested changes. LMK if anything remains unclear.